components/style/values/specified/length.rs

```diff
@@ -701,7 +688,7 @@ pub type NonNegativeLengthOrNormal = Either<NonNegativeLength, Normal>;
 pub type NonNegativeLengthOrAuto = Either<NonNegativeLength, Auto>;
 
 /// Either a NonNegativeLength or a NonNegativeNumber value.
```

Comment:
Please mention that the order here is important so `0` parses as a number instead of a length?